packages/react-aria-components/stories/ListBox.stories.tsx

```diff
@@ -187,3 +190,74 @@ export const ListBoxHover = () => (
   </ListBox>
 );
 
+export function VirtualizedListBox() {
```

Comment:
I noticed that if the focused item is scrolled out of view, arrow key navigating won't scroll the newly focused item into view.